rest/changes_api.go

```diff
@@ -323,16 +326,35 @@ func (h *handler) sendChangesForDocIds(userChannels base.Set, explicitDocIds []s
 		}
 
 		doc.RevID = body["_rev"].(string)
-		doc.Sequence = sequence
+		doc.Sequence = populatedDoc.Sequence
```

Comment:
Is there a reason to mutate the incoming doc here?  Can we work with populatedDoc/body instead?